code/edge/src/test/java/com/adobe/marketing/mobile/EdgePublicAPITests.java

```diff
@@ -55,19 +51,6 @@ public void tearDown() {
 		mockCore.close();
 	}
 
-	@Test
-	public void testExtensionVersion_verifyModuleVersionInPropertiesFile_asEqual() {
-		Properties properties = loadProperties(GRADLE_PROPERTIES_PATH);
```

Comment:
Looks like the `loadProperties` function is only used here. Can you remove it from this file?